### ISSUE TITLE
StickyScroll Not working properly for php

### ISSUE THREAD

**Issue by webdesires** [microsoft/vscode#160691]
Does this issue occur when all extensions are disabled?: Yes - in fact doesnt work at all...

- VS Code Version: 1.71.0
- OS Version: Windows 11 & Windows 10

Steps to Reproduce:

1. Open a php class file or plain php file of functions.
2. scroll, and the class will get sticky but no functions you are in will go sticky.
Here is a short video.
https://webdesires.dev/u/120922/Code_8Sw1nbpY2c.mp4

You will see the class will go sticky, but as you scroll no functions in the class will go sticky.
If you use a plain functions file nothing goes sticky...

**Comment by gjsjohnmurray** [comment]
You need a PHP extension which is able to populate the Outline view of VS Code.

**Comment by webdesires** [comment]
Which PHP extension? I already have a few...

https://webdesires.dev/u/120922/Code_2YdIosqrKh.png

The intelephense is disabled since it was dropping errors all over my code as it seems to only want to work if you use projects I guess... Which I dont use.

**Comment by webdesires** [comment]
Outline works fine btw... https://webdesires.dev/u/120922/Code_18D4l8rO6M.png

**Comment by webdesires** [comment]
Ok I fixed my own issue... It seems to be a problem with the plugin PHP symbols (https://marketplace.visualstudio.com/items?itemName=linyang95.php-symbols).

I noticed firstly that on the Outline although working the "follow cursor" was not working at all.

Disabled this plugin and installed PHP IntelliSense (https://marketplace.visualstudio.com/items?itemName=zobo.php-intellisense) and suddenly not only is outline working properly but also sticky is working!
